### ISSUE TITLE
Bug Report:  Missing "alt" Property for Images in resources.html file 

### ISSUE THREAD

**Issue by GauriBhandari** [SauravMukherjee44/Aec-Library-Website#1455]
### Reproduction steps

I noticed that some images on the website are missing the "alt" attribute. The "alt" attribute is essential for accessibility as it provides a text alternative to the images. This allows screen readers and other assistive technologies to describe the image to users who may have visual impairments.

Steps to Reproduce:

1.Navigate to resources.html.
2.Inspect the images on the page and look for the "alt" attribute.

### Expected behavior

All images on the website should have a descriptive and meaningful "alt" attribute that conveys the content and purpose of the image.

### Actual Behavior

The website's images are missing a attribute called "alt," which affects how people with visual impairments can access the content.

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/SauravMukherjee44/Aec-Library-Website/blob/main/CODE_OF_CONDUCT.md)

**Comment by rohanbabbar04** [comment]
Sure go for it
